modules/mod_comment/mod_comment.erl

```diff
@@ -84,6 +86,15 @@ event(#submit{message={newcomment, Args}, form=FormId}, Context) ->
             Context
     end.
 
+get_rating(Context) ->
+    case z_context:get_q(<<"rating">>, Context) of
+        undefined -> undefined;
+        RatingString -> 
+            Rating = z_convert:to_integer(RatingString),
+            true = 0 =< Rating andalso Rating =< 5,
+            Rating
+     end.
+        
 
 %% @doc Return the list of recent comments.  Returned values are the complete records.
 observe_search_query(#search_query{search={recent_comments, []}, offsetlimit=OffsetLimit}, Context) ->
```

Comment:
Line 100 is too long: observe_search_query(#search_query{search={recent_comments, []}, offsetlimit=OffsetLimit}, Context) ->.